cmd/get.go

```diff
@@ -50,11 +51,21 @@ and provides a fast and easy alternative to a package manager.`,
 	command.Flags().StringP("output", "o", "", "Output format of the list of tools (table/markdown/list)")
 	command.Flags().Bool("stash", true, "When set to true, stash binary in HOME/.arkade/bin/, otherwise store in /tmp/")
 	command.Flags().StringP("version", "v", "", "Download a specific version")
+	command.Flags().Bool("dry-run", false, "When set to true, performs a dry run of binary download and install location")
 
 	command.RunE = func(cmd *cobra.Command, args []string) error {
+		dry_run, _ := command.Flags().GetBool("dry-run")
```

Comment:
Variables should be written `dryRun` in Go, this looks more like a Ruby / Python / PHP variable name.